python/bin/mob

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env python3
+from mobilecoin import CommandLineInterface
```

Comment:
I think ```mob``` is fine. The mob prompt isn't a command line tool or part of path, whereas this will be. When a user wants to use the mob builder tool, they have to explicitly run it from the directory in the format of ```./mob prompt```, which should override any path variable, I believe.

Reply:
If it's really an issue for people who use both tools, then they can easily rename the binary to whatever is convenient

Reply:
Okay, moved it back to just ./mob for the builder image.